src/index.js

```diff
@@ -144,21 +163,23 @@ export default class RNPickerSelect extends PureComponent {
     }
 
     onUpArrow() {
+        const { onUpArrow } = this.props;
+
         this.togglePicker();
-        setTimeout(() => {
-            this.props.onUpArrow();
-        });
+        setTimeout(onUpArrow);
```

Comment:
This is a good time to add a comment why setTimeout is on these two arrow methods imo